Gemfile.jruby-2.3.lock.release

```diff
@@ -420,13 +420,13 @@ GEM
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       mail (~> 2.6.3)
       mime-types (< 3)
-    logstash-output-file (4.1.1)
+    logstash-output-file (4.2.1)
```

Comment:
New write_behavior feature. Value can be "append" (default) or "overwrite". If "append", events will be appended to the end of the file. If "overwrite", the file will only contain the last event written.

Reply:
I see no reason to use this new minor on a .patch release